unit_test/learning_mode_test.cc

```diff
@@ -26,7 +38,7 @@ BOOST_AUTO_TEST_CASE(learning_mode_online) {
 }
 
 BOOST_AUTO_TEST_CASE(learning_mode_other) {
-  const char* imitation = "other";
-  auto mode = reinforcement_learning::learning::to_learning_mode(imitation);
+  const char* apprentice = "other";
```

Comment:
I cannot remember why I put `imitation` before but I think this should be `other`

Reply:
renamed.